mypy-check

```diff
@@ -4,15 +4,7 @@ set -evu
 # single argument = root directory of module to test
 # Requires mypy >= 0.800
 
-MODULEDIR=$1
-
-# set MYPYPATH
-MYPYPATH="."
-for MOD in $(ls -d "$MODULEDIR"*); do
-    MOD_PATH="$(cd "$MOD" && pwd)"
-    MYPYPATH="$MYPYPATH:$MOD_PATH"
-done
-export MYPYPATH="$MYPYPATH"
+MYPYPATH=$1
```

Comment:
This is something we should change on the other repos, too?

Reply:
Yes, probably.